app/templates/main/index.html

```diff
@@ -4,21 +4,122 @@
+        }
+      }
+    </script>
+
     <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
     <meta charset="utf-8">
     <meta name="csrf-token" content="{{ csrf_token() }}">
 
     <div id="map-list-grid" class="ui two column horizontally padded grid">
       <div class="four wide column">
-        <form id="resources-form">
-            <input type="text" id="resources-input"
-                   placeholder="Enter a resource keyword">
-        </form>
-        <input id="pac-input" type="text"
-               placeholder="Enter a location">
-        <div id="list" class="ui middle aligned divided list">
+        <div id="search">
+            <form id="resources-form">
+               <div class="ui fluid icon input">
+                <i class="search icon"></i>
+                <input type="text" id="resources-input" class="controls"
+                       placeholder="Enter a resource keyword">
+                </div>
+            </form>
+
+          <div class="ui fluid icon input" id="location-search">
+            <i class="search icon"></i>
+            <input type="text" id="pac-input" class="controls"
+                   placeholder="Enter a location">
+          </div>
+
+          <div id="default-search" class="option-group">
+            <div class="descriptor-group">
+              <h5>DESCRIPTOR #1</h5>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option" class="checkbox-option">
+                        <label>Option ekkeke</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+            </div>
+          </div>
+
+          <div id="advanced-search" class="option-group-advanced" style="display:none;">
```

Comment:
instead of setting style to be `display:none` you can put in JS to hide it initially (see above jQuery comment)